tlsconfig/certpool_other.go

```diff
@@ -0,0 +1,11 @@
+// +build !go1.7
+
+package tlsconfig
+
+import "crypto/x509"
+
+// SystemCertPool returns an new empty cert pool,
+// accessing system cert pool is supported in go 1.7
+func SystemCertPool() *x509.CertPool {
+	return x509.NewCertPool()
```

Comment:
It's possible if we change the interface to return an error, we can just return an error here when not on windows let the caller decide whether to use `x509.NewCertPool()` instead. At least in that case we can have output in the debug logs that the pool was not merged with the system pool.